src/components/Comments.jsx

```diff
@@ -0,0 +1,36 @@
+import { useEffect, useState } from "react"
+import { useParams } from "react-router-dom"
+import { getComments } from "../utils/api"
+
+const Comments = () => {
+const {review_id} = useParams()
+const [comments, setComments] = useState([])
+const [isLoading, setIsLoading] = useState(true)
+
+useEffect(() => {
+    setIsLoading(true)
+    getComments(review_id).then((commentById) => {
+        setComments(commentById)
+        setIsLoading(false)
+    })
+}, [review_id])
+
```

Comment:
Your comment section could benefit from a message that appears when there are zero comments, like something that says "No one has commented yet"